examples/markdown/src/bin/example_markdown_test_server.rs

```diff
@@ -0,0 +1,19 @@
+#![cfg(not(target_arch = "wasm32"))]
```

Comment:
Is this server only needed for test?

It's confusing how to use the example. Can you please add a README file, explaining how to run the example and the purpose of this server and it's relation with `trunk serve`?